pkg/cmd/roachtest/run.go

```diff
@@ -487,6 +487,9 @@ func maybeEmitDatadogEvent(
 	details := fmt.Sprintf("cluster: %s\n", clusterName)
 	ev := statsd.NewEvent(fmt.Sprintf("op %s %s", opSpec.Name, status), details)
 
+	ev.Tags = append(ev.Tags, fmt.Sprintf("operation-name:%s", opSpec.Name))
+	ev.Tags = append(ev.Tags, fmt.Sprintf("operation-status:%s", status))
```

Comment:
I think this is not required, `ev.AlertType` is already giving the status.

Reply:
It is better to have a separate tag for`operation-status` then relying on `ev.AlertType`. Because for  `eventType=eventOpStarted` and `eventType=eventOpFinishedCleanup` the value of `ev.AlertType is Info` but value of `operation-status` is different for these eventType. Given as of now we are not sure how we want to leverage this to configure datadog monitor better to have both alertType and operation-status.